agent/src/main/java/io/keploy/advice/ksql/RegisterDriverAdvice.java

```diff
@@ -37,16 +36,19 @@ public static void enterMethod(@Advice.Origin Method method, @Advice.Argument(va
                     break;
                 default:
                     System.out.println("Dialect for driver: " + driverClassName + " is not supported yet");
+                    driverClassName = KDriver.class.getName();
+                    break;
             }
+        } else {
+            driverClassName = KDriver.class.getName();
         }
-        driverClassName = "io.keploy.ksql.KDriver";
     }
 
     /**
-     * This method gets executed after the setDriverClassName method of DataSourceProperties class.This does nothing as we don't
```

Comment:
Actually u can remove the whole method of @Advice.OnMethodExit as it is not used.